tests/aggregation/tests.py

```diff
@@ -151,6 +151,10 @@ def test_aggregate_alias(self):
         vals = Store.objects.filter(name="Amazon.com").aggregate(amazon_mean=Avg("books__rating"))
         self.assertEqual(vals, {'amazon_mean': Approximate(4.08, places=2)})
 
+    def test_aggregate_transform(self):
+        vals = Store.objects.aggregate(min_month=Min('original_opening__month'))
+        self.assertEqual(vals, {'min_month': 3})
+
```

Comment:
It'd be great if you could add a test that also performs a `JOIN` followed by a transform as the change code involves work in `setup_join` as well but this single test doesn't spawn a `JOIN` at all.

Reply:
An earlier version of this commit didn't include that change and triggered several test failures/errors. I can add an extra test for it anyway if you think that's necessary.

Reply:
yeah I think it would be worthwhile to at least test a single JOIN scenario.